packages/angular/cli/README.md

```diff
@@ -234,26 +234,33 @@ For more informations about Node.js debugging in VS Code, see the related [VS Co
 
 In order to investigate performance issues, CPU profiling is often useful.
 
-To capture a CPU profiling, you can:
+#### Creating a profile
 
-1. install the v8-profiler-node8 dependency: `npm install v8-profiler-node8 --no-save`
-1. set the NG_CLI_PROFILING Environment variable to the file name you want:
-   - on Unix systems (Linux & Mac OS X): ̀`export NG_CLI_PROFILING=my-profile`
-   - on Windows: ̀̀`setx NG_CLI_PROFILING my-profile`
+To capture a CPU profiling, you run the build with `NG_BUILD_PROFILING` environment variable set to `1` or `true`
 
-Then, just run the ng command on which you want to capture a CPU profile.
-You will then obtain a `my-profile.cpuprofile` file in the folder from which you ran the ng command.
+- on Unix systems (Linux & Mac OS X): ̀`NG_BUILD_PROFILING=1 ng build`
+- on Windows: ̀̀`set NG_BUILD_PROFILING=1 && ng build`
+
+You will then obtain a `chrome-profiler-events.json` file in the folder from which you ran the `ng` command.
+
+Or Node.js 16+ users can use the Node.js command line argument `--cpu-prof` to create a CPU profile.
+
+```bash
+node --cpu-prof node_modules/.bin/ng build
+```
+
+In addition to these two methods, another, more elaborated way to capture a CPU profile using the Chrome Devtools is detailed in https://github.com/angular/angular-cli/issues/8259#issue-269908550.
+
+#### Opening a profile
 
 You can use the Chrome Devtools to process it. To do so:
 
 1. open `chrome://inspect/#devices` in Chrome
 1. click on "Open dedicated DevTools for Node"
 1. go to the "profiler" tab
-1. click on the "Load" button and select the generated .cpuprofile file
+1. click on the "Load" button and select the generated `.cpuprofile` file
```

Comment:
As a reader, I thought we were generating a `chrome-profiler-events.json` file? How do we visualize that file? Where did this `.cpuprofile` file come from? Was that from the GitHub comment linked above?

Reply:
Either that or using the `—cpuprof` argument. `chrome-profiler-events.json` Is only generated when using the env variable.

Reply:
I'm a little confused by your other comment that `--cpu-prof` generates the same thing, but `chrome-profiler-events.json` is only generated with the env variable. Is it that they are the same type and can be visualized the same way in DevTools, but have different content? What exactly si the difference between these files?

I think it would be helpful to more clear about what is generating each of these files and how each of them can be visualized (whether the same means for both or something different).

Reply:
Both generate CPU profiles, but with different file names, the `--cpu-prof` is a Node.Js option which we don’t have a control over and therefore we cannot control the filename like we do with the `NG_BUILD_PROFILING` which generates a cpu profile via a webpack plugin https://github.com/angular/angular-cli/blob/0b6071af3a51e7d3f38a661bd4e0a3c3e81aff2f/packages/angular_devkit/build_angular/src/webpack/configs/common.ts#L138.